GameDataParser/Files/Export/ItemMetadataExport.cs

```diff
@@ -9,11 +9,13 @@ namespace GameDataParser.Files.Export
 {
     public class ItemMetadataExport
     {
+        private readonly SetPath setPath= new SetPath();
```

Comment:
Space before the `=`?

Reply:
Also, if this is only used in the `Export()` method, why is it defined outside of it?

Reply:
You're right this could be ask without calling it outside and set the class to static